IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -1035,8 +1069,28 @@ - (BOOL)saveAccount:(MSIDAccount *)account
     
     for (MSIDAccountCacheItem *accountCacheItem in allAccounts)
```

Comment:
Should this be invoked after we query legacy accounts? Otherwise, if an account is marked as signed out in account metadata cache, but is also present in legacy cache, it will still be returned.

Reply:
Discussed offline. Now code is changed to do filter for legacy accounts as well :)